datamodel/initdb.d/09_metadata.sql

```diff
@@ -0,0 +1,14 @@
+-- Assumes the PSQL client
```

Comment:
Why is it called 09? I thought 07 was the next slot...

Reply:
Derp, must've mistyped...

Reply:
@HenrikHL fixed